policyengine_us/variables/gov/states/ri/tax/income/adjusted_gross_income/social_security/ri_social_security_modification.py

```diff
@@ -0,0 +1,65 @@
+from policyengine_us.model_api import *
+import numpy as np
+
+
+class ri_social_security_modification(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island Social Security Modification"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/Social%20Security%20Worksheet_w.pdf"
+    defined_for = "ri_social_security_modification_eligibility"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        is_head = person("is_tax_unit_head", period)
+        is_spouse = person("is_tax_unit_spouse", period)
+        age = person("age", period)
+        birth_year = period.start.year - age
+
+        p = parameters(
+            period
+        ).gov.states.ri.tax.income.adjusted_gross_income.subtractions.social_security
+
+        # Age-based eligibility.
+        age_conditions = birth_year <= p.birth_date_limit
```

Comment:
can we delete this as it is part of the eligiblity?

Reply:
I think we need to include this to specify which one (head or spouse)'s social security amount is used.